Demo/Color+Helpers.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+public extension UIColor {
```

Comment:
Is probably better to keep it as lean as possible. Use plain RGB colors for the demo.